projects/website/.env

```
@@ -0,0 +1,3 @@
+VITE_APP_ID="${APP_ID}"
+VITE_UNHEAD_HOST="${HOST}"
+VITE_ESPACE_WEBSITE_HOST="${ESPACE_WEBSITE_HOST}"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)